src/test/java/cloud/eppo/ConfigurationRequestorTest.java

```diff
@@ -21,7 +22,7 @@ public class ConfigurationRequestorTest {
 
   @Test
   public void testInitialConfigurationFuture() throws IOException {
-    IConfigurationStore configStore = new ConfigurationStore();
+    IConfigurationStore configStore = Mockito.spy(new ConfigurationStore());
```

Comment:
in place of checking for a `null` configuration, spy on the config store and verify calls to `saveConfiguration` in addition to checking for empty